src/com/lftechnology/java_training/dec31/EmployeeTest.java

```diff
@@ -0,0 +1,123 @@
+package com.lftechnology.java_training.dec31;
+
+import java.util.Scanner;
+
+/**
+ * Write a test application named EmployeeTest that demonstrates class
+ * Employee's capabilities. Create two Employee objects and display each
+ * object's full name and yearly salary. Then give each Employee a 10% raise and
+ * display each Employee's full name and yearly salary again.
+ * 
+ * @author utsabn34
+ * 
+ */
+public class EmployeeTest {
+
+	public static void main(String[] args) {
+		Employee employee1 = new Employee();
+		Employee employee2 = new Employee();
+		employee1.assignData("employee1");
+		employee2.assignData("employee2");
+		employee1.displayData();
+		employee2.displayData();
+		System.out.println("After 10% rise in Salary:");
+		employee1.setData(employee1.getFirstName(), employee1.getLastName(),
+				employee1.getIncSalary());
+		employee2.setData(employee2.getFirstName(), employee2.getLastName(),
+				employee2.getIncSalary());
+		employee1.displayData();
+		employee2.displayData();
+	}
+
+}
+
+/**
+ * * Create a class called Employee that includes three pieces of information as
+ * instance variables a first name (type String), a last name (type String) and
+ * a monthly salary (double). Your class should have a constructor that
+ * initializes the three instance variables. Provide a set and a get method for
+ * each instance variable.
+ * 
+ * @author utsabn34
+ * 
+ */
+class Employee {
+	private String fName = "";
+	private String lname = "";
+	private double salary = 0;
+	private Scanner input = new Scanner(System.in);
+
+	/**
+	 * set value to instance variable
+	 * 
+	 * @param fName
+	 * @param lName
+	 * @param salary
```

Comment:
If you are going to give parameter in the method description, give proper meaning. Like fName: first name of an employee